server.py

```diff
@@ -1,4 +1,5 @@
 from flask import Flask, render_template
+from flask import request,jsonify

```

Comment:
Please, respect PEP8.
```suggestion
from flask import request, jsonify
```
You can also install Anaconda to SublimeText3 in order to enforce the codestyle.